docs/StyleGAN2-ADATraining.md

```diff
@@ -48,7 +48,7 @@ Note that there is a bug in the image generation code when using grey-scale imag
     ```
 
 ## Prepare the data
-The set of images used to train the model must be in the correct form for use with StyleGAN2-ADA.  Its `dataset_tool.py` script is used to take a set of images, format, and output them to a directory.  The source images were taken from those created according to the [Prepare Image Set For StyleGAN models](https://github.com/inclusive-design/baby-bliss-bot/blob/8ff76322918af4b187708cda2a87b8bdb67271b5/docs/PrepareImageSetForStyleGAN.md) document, from [PR #6](https://github.com/inclusive-design/baby-bliss-bot/pull/6). (Aside: As of this writing, 15-Jun-2023, the image preparation documentation is on a branch.  This link needs to update when that branch is merged with main).  These were gathered into a tar file and processed by an SBatch shell script to create a tar file of prepared images.  In summary:
+The set of images used to train the model must be in the correct form for use with StyleGAN2-ADA.  Its `dataset_tool.py` script is used to take a set of images, format, and output them to a directory.  The source images were taken from those created according to the [Prepare Image Set For StyleGAN models](https://github.com/inclusive-design/baby-bliss-bot/blob/main/docs/PrepareImageSetForStyleGAN.md) document.  These were gathered into a tar file and processed by an SBatch shell script to create a tar file of prepared images.  In summary:
```

Comment:
Minor:  the link to the "Prepare Image Set ..." documentation could be relative, like:  `(./PrepareImageSetForStyleGAN.md)`.

Change as you see fit.